scripts/wrappers/gatk/testWDLTasks.py

```diff
@@ -0,0 +1,21 @@
+#!/bin/python
+import subprocess
+import sys
+import os
+
+#To use: pass in the directory containing all generated WDL tasks as the first argument
```

Comment:
You could make this shorter and unambiguous by giving an example command, eg

    testWDLTasks.py path/to/wdl_tasks_dir /path/to/wdltool.jar